packages/api-explorer/src/index.jsx

```diff
@@ -254,7 +257,11 @@ ApiExplorer.propTypes = {
   i18n: PropTypes.shape({
     locale: PropTypes.string,
     defaultLocale: PropTypes.string,
-  }), 
+  }),
+  showOnlyAPI: PropTypes.bool,
+  defaultOpen: PropTypes.bool,
+  defaultOpenDoc: PropTypes.string,
+  onChange: PropTypes.func,
```

Comment:
More declarative to what is changed with this prop

Reply:
Named it `onDocChange`.